pkg/svcgen/params/builder.go

```diff
@@ -0,0 +1,275 @@
+package params
+
+import (
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/izumin5210/grapi/pkg/cli"
+	"github.com/izumin5210/grapi/pkg/grapicmd/util/fs"
+	"github.com/pkg/errors"
+	"github.com/serenize/snaker"
+)
+
+type Builder interface {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Builder should have comment or be unexported